libc/test/src/math/exp10m1f_test.cpp

```diff
@@ -80,7 +80,7 @@ TEST_F(LlvmLibcExp10m1fTest, InFloatRange) {
   constexpr uint32_t STEP = UINT32_MAX / COUNT;
   for (uint32_t i = 0, v = 0; i <= COUNT; ++i, v += STEP) {
     float x = FPBits(v).get_val();
-    if (isnan(x) || isinf(x))
+    if (FPBits(v).is_nan() || FPBits(v).is_inf())
```

Comment:
Could just use `.is_inf_or_nan()`.

```suggestion
    if (FPBits(v).is_inf_or_nan())
```